build.sbt

```
@@ -27,42 +20,12 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
   .in(file("core"))
   .settings(
     name := "cats-uri",
-
     libraryDependencies ++= Seq(
-      "org.typelevel"               %% "cats-core"                  % catsV,
-      "org.typelevel"               %% "cats-effect"                % catsEffectV,
-
-      "co.fs2"                      %% "fs2-core"                   % fs2V,
-      "co.fs2"                      %% "fs2-io"                     % fs2V,
-
-      "org.http4s"                  %% "http4s-dsl"                 % http4sV,
-      "org.http4s"                  %% "http4s-ember-server"        % http4sV,
-      "org.http4s"                  %% "http4s-ember-client"        % http4sV,
-      "org.http4s"                  %% "http4s-circe"               % http4sV,
-
-      "io.circe"                    %% "circe-core"                 % circeV,
-      "io.circe"                    %% "circe-generic"              % circeV,
-      "io.circe"                    %% "circe-parser"               % circeV,
-
-      "org.tpolecat"                %% "doobie-core"                % doobieV,
-      "org.tpolecat"                %% "doobie-h2"                  % doobieV,
-      "org.tpolecat"                %% "doobie-hikari"              % doobieV,
-      "org.tpolecat"                %% "doobie-postgres"            % doobieV,
-
-      "org.typelevel"               %%% "munit-cats-effect-3"        % munitCatsEffectV         % Test,
-
+      "org.typelevel" %% "cats-core" % V.catsV
     )
-  ).jsSettings(
-    scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule)},
+  )
+  .jsSettings(
+    scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
   )
 
-lazy val site = project.in(file("site"))
-  .disablePlugins(MimaPlugin)
-  .enablePlugins(DavenverseMicrositePlugin)
-  .dependsOn(core.jvm)
-  .settings{
-    import microsites._
-    Seq(
-      micrositeDescription := "A URI with Cats instances",
-    )
-  }
+lazy val site = project.in(file("site")).disablePlugins(MimaPlugin).dependsOn(core.jvm)
```

```suggestion
lazy val site = project.in(file("site")).enablePlugins(TypelevelSitePlugin).dependsOn(core.jvm)
```

You'll also need to add a dependency to `sbt-typelevel-site`.